src/main/scala/mesosphere/marathon/upgrade/TaskReplaceActor.scala

```diff
@@ -26,7 +26,9 @@ class TaskReplaceActor(
 
   val tasksToKill = taskTracker.marathonAppTasksSync(app.id)
   val appId = app.id
-  val version = app.version.toString
+  val version = app.version
+  // FIXME: Remove!
```

Comment:
Please mark as FIXME(PK) and create a ticket for you to remove all occurrences of FIXMEs.